### DISCUSSION TITLE
Swarm of Lima VM's

### DISCUSSION THREAD

**Discussion by kdowney-lot49** [lima-vm/lima#2431] [Q&A]
From within a Lima VM, is there a way to control Lima on the host, e.g. create / start / stop other VM's?

The use case is this: VSCode dev container running in Lima VM on ARM64; one or more test VM's running x86_64 with Rosetta enabled. I want to see if there's a way I could mount Lima's socket into the dev container and reach back to Lima, kind of like the existing VSCode Docker-from-Docker but via Lima.

**Comment by afbjorklund** [comment]
There is currently no remote control of the limactl, so you would have to ssh from the VM to the host.
